style.css

```diff
@@ -53,7 +54,35 @@ body {
 .search {
   grid-area: search;
   padding-right: 100px;
-  /* border: 2px solid rgb(29, 26, 226); */
+  border: 2px solid rgb(226, 225, 225);
+  background-color: rgb(226, 225, 225);
+  border-radius: 30%;
+  text-align: center;
+  text-decoration: black;
+  justify-content: center;
+}
```

Comment:
Search doesn't look right.  Work needed to get colour better & search looking more accurate.